src/input/plugins/CurlInputPlugin.cxx

```diff
@@ -431,9 +434,11 @@ CurlInputStream::InitEasy()
 		request->SetOption(CURLOPT_PROXYUSERPWD,
 				   StringFormat<1024>("%s:%s", proxy_user,
 						      proxy_password).c_str());
+        if (cacert != nullptr)
+            request->SetOption(CURLOPT_CAINFO, cacert);
```

Comment:
So CURL calls this CAINFO and you call it cacert... and it's undocumented!

Reply:
https://curl.se/libcurl/c/CURLOPT_CAINFO.html
CURLOPT_CAINFO - path to Certificate Authority (CA) bundle 
and from example 
curl_easy_setopt(curl, CURLOPT_CAINFO, "/etc/certs/cabundle.pem");

Do you think I should use another name? I have selected cacert because .curlrc has option cacert for CA bundle